website/source/docs/agent/config-entries/proxy-defaults.html.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: "docs"
+page_title: "Configuration Entry Kind: Proxy Defaults"
+sidebar_current: "docs-agent-cfg_entries-proxy_defaults"
+description: |-
+  Proxy defaults allow for configuring global config defaults across all services for Connect proxy configuration. Currently, only one global entry is supported.
+---
+
+# Proxy Defaults - `proxy-defaults`
```

Comment:
```suggestion
# Proxy Defaults - `proxy-defaults`
```

Reply:
Kinds in the headings look a bit weird. May want to just explicitly mention them in the intro paragraphs instead

Reply:
Rearranged it to not repeat the kind in the heading.